mobile-app/app/screens/PlaygroundNavigator/sections/PlaygroundOperations.tsx

```diff
@@ -0,0 +1,71 @@
+import { useDispatch } from 'react-redux'
+import { View } from '@components'
+import { PlaygroundTitle } from '@screens/PlaygroundNavigator/components/PlaygroundTitle'
+import { useWalletContext } from '@shared-contexts/WalletContext'
+import { useWhaleApiClient } from '@shared-contexts/WhaleContext'
+import { usePlaygroundContext } from '@contexts/PlaygroundContext'
+import { useEffect, useState } from 'react'
+import { WalletAddressIndexPersistence } from '@api/wallet/address_index'
+import { PlaygroundAction } from '@screens/PlaygroundNavigator/components/PlaygroundAction'
+import { fetchTokens } from '@store/wallet'
+
+export function PlaygroundOperations (): JSX.Element {
+  const { wallet } = useWalletContext()
+  const client = useWhaleApiClient()
+  const dispatch = useDispatch()
+  const {
+    api,
+    rpc
+  } = usePlaygroundContext()
+  const [status, setStatus] = useState<string>('loading')
+
+  useEffect(() => {
+    api.wallet.balances().then(() => {
+      setStatus('online')
```

Comment:
can we replace the status string with enum or type?

Reply:
Updated